composer.json

```diff
@@ -19,7 +19,8 @@
         "phpunit/phpunit": "^9.5",
         "bamarni/composer-bin-plugin": "^1.4",
         "symfony/process": "^5.2",
-        "phpstan/phpstan": "^1.2"
+        "phpstan/phpstan": "^1.2",
+        "symfony/var-dumper": "^6.2"
```

Comment:
allows the use of `dump($foo)` and `dd($foo)` - much better than `var_dump` for debugging